mkosi/resources/mkosi.md

```diff
@@ -1254,6 +1254,21 @@ boolean argument: either `1`, `yes`, or `true` to enable, or `0`, `no`,
   Additionally, the suffixes `K`, `M` and `G` can be used to specify a
   size in kilobytes, megabytes and gigabytes respectively.
 
+## Specifiers
+
+The current value of various settings can be accessed when parsing
+configuration files by using specifiers. The following specifiers are
+understood:
+
+| Setting            | Specifier |
+|--------------------|-----------|
+| `Distribution=`    | `%d`      |
+| `Release=`         | `%r`      |
+| `Architecture=`    | `%a`      |
+| `Format=`          | `%t`      |
+| `Output=`          | `%o`      |
+| `OutputDirectory=` | `%O`      |
```

Comment:
```suggestion
| `OutputDirectory=` | `%O`      |
| Literal `%`        | `%%`      |
```
so that people know how to do that. It's also a bit of a backward compat break we should mention in the news, but I don't think people commonly have percent signs in their paths. At least I hope so. :)